.agents/modernize-explainer.md

```
@@ -0,0 +1,466 @@
+| Qwiq.Linq | ✅ | ❌ | ✅ |
+| Qwiq.Mapper | ✅ | ❌ | ✅ |
+| Qwiq.Identity | ✅ | ❌ | ✅ |
+| Qwiq.Identity.Soap | ✅ | ❌ | ❌ |
+| Qwiq.Linq.Identity | ✅ | ❌ | ✅ |
+| Qwiq.Mapper.Identity | ✅ | ❌ | ✅ |
+| Qwiq.Mocks | ✅ | ❌ | ✅ |
+| Qwiq.Tests.Common | ✅ | ❌ | ✅ |
+| Qwiq.Core.Tests | ✅ | ❌ | ✅ |
+| Qwiq.Linq.Tests | ✅ | ❌ | ✅ |
+| Qwiq.Mapper.Tests | ✅ | ❌ | ✅ |
+| Qwiq.Identity.Tests | ✅ | ❌ | ✅ |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)